setup.py

```diff
@@ -110,8 +110,12 @@ def find_rel_test():
     provides=["eb"] + easybuild_packages,
     test_suite="test.framework.suite",
     zip_safe=False,
-    install_requires=["vsc-base >= 2.2.6"],
+    install_requires=[
+        'setuptools',
```

Comment:
this should specify a minimal version

cfr. https://github.com/hpcugent/vsc-install/pull/31